src/plugins/intel_cpu/src/config.h

```diff
@@ -49,6 +49,8 @@ struct Config {
     size_t rtCacheCapacity = 5000ul;
     InferenceEngine::IStreamsExecutor::Config streamExecutorConfig;
     InferenceEngine::PerfHintsConfig  perfHintsConfig;
+    bool use_ht_value = true;
```

Comment:
Please use camleCase and rename to `useHyperThreading`

Reply:
updated